src/cli/main.py

```diff
@@ -27,23 +33,37 @@ def getting_started(name: str):
 @app.command()
 def build_skill_assessment(name: str, skills: str = ''):
     "Builds skill assessment using standards passed."
-    print(f"Building Standards in /standards folder {name}")
+    print(f"Building {name} assesment in /assessment/user folder")
     # TODO standards index
     # Open Standards
-    file_and_path = Path("tests/data") / f"{name}.json"
-    skill_matrix_overview_path = "cli/templates/skill_matrix_overview.md"
+    if environment == 'production':
+        file_and_path_string = "standards/" + name + ".json"
+    else:
+        file_and_path_string = "src/cli/tests/data/" + name + ".json"
+
+    # file_and_path = Path("tests/data") / f"{name}.json"
+    file_and_path = Path(file_and_path_string)
+    skill_matrix_overview_path = "cli/templates/skill_matrix_overview.md" 
```

Comment:
Could this be a Path object right away?

Reply:
Good notes on these. We are moving the constructor up for dev and production.